src/main/kotlin/com/openlattice/mechanic/upgrades/MigrateOrgPermissionsUpgrade.kt

```diff
@@ -0,0 +1,127 @@
+package com.openlattice.mechanic.upgrades
+
+import com.hazelcast.query.Predicates
+import com.openlattice.authorization.Ace
+import com.openlattice.authorization.AceKey
+import com.openlattice.authorization.AceValue
+import com.openlattice.authorization.Acl
+import com.openlattice.authorization.Action
+import com.openlattice.authorization.mapstores.PermissionMapstore
+import com.openlattice.authorization.securable.SecurableObjectType
+import com.openlattice.hazelcast.HazelcastMap
+import com.openlattice.mechanic.Toolbox
+import com.openlattice.postgres.external.ExternalDatabaseConnectionManager
+import com.openlattice.postgres.external.ExternalDatabasePermissioningService
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
+import java.sql.SQLException
+import java.util.UUID
+import java.util.regex.Pattern
+
+class MigrateOrgPermissionsUpgrade(
+        toolbox: Toolbox,
+        private val exConnMan: ExternalDatabaseConnectionManager,
+        private val exDbPermMan: ExternalDatabasePermissioningService
+): Upgrade {
+
+    val logger: Logger = LoggerFactory.getLogger(MigrateOrgPermissionsUpgrade::class.java)
+
+    private val entitySets = HazelcastMap.ENTITY_SETS.getMap(toolbox.hazelcast)
+    private val externalTables = HazelcastMap.EXTERNAL_TABLES.getMap(toolbox.hazelcast)
+    private val permissions = HazelcastMap.PERMISSIONS.getMap(toolbox.hazelcast)
+
+    override fun upgrade(): Boolean {
+        val uuidRegex= Pattern.compile(
+            "[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}",
+            Pattern.CASE_INSENSITIVE
+        )
+
+        // filtering for a specific org
+        println("Organization to filter: ")
+        val input = readLine()
```

Comment:
I don't think it's a good idea to do `readLine()` because

1. it forces the caller to have to wait until everything has booted to pass the org id, which can sometimes take up to 15 minutes
2. it prevents the caller from running the migration in the background. I'm sure there's ways around this but still... It would be easier to just hardcode the org id and recompile.

With that in mind, maybe we should just get rid of all the "pass an org id" stuff and just hardcode it 🤔

Reply:
Yeah, the `readline()` is only so that we don't have to recompile every time we want to perform the migration on a different org. It's either that, or having to rewrite the mechanic upgrade logic a little. 

I do agree with your points, so yeah I will get rid of the passing org id stuff and switch back to hardcoding it.